src/ru/geekbrains/lesson6/Animal.java

```diff
@@ -0,0 +1,19 @@
+package ru.geekbrains.lesson6;
+
+public abstract class Animal {
+
+    protected String name;
+
+    public Animal (String name){
+        this.name = name;
+    }
+
+    public abstract void run(int i);
+
+    public abstract void swim(int i);
+
+    public abstract void jump(double i);
```

Comment:
```suggestion
    public abstract void jump(double height);
```
лучше сразу привыкать давать осмысленные имена всему по коду